konfera/register/views.py

```diff
@@ -21,6 +21,10 @@ def register_volunteer(request, slug):
             new_ticket.type = volunteer_ticket_type
             new_ticket.save()
 
+            if volunteer_ticket_type.accessibility != 'public':
```

Comment:
It seems, I havent explained the issue very well. What I meant was that the initial check in this file (get_object_or_404 should have one more search attribute accessibility). Eg. current setup would also allow volunteer registration event if there  is only private ticket, you update so it will only allow if there is public ticket.

Your change would modify existing ticket from private to public... Please revert back, other thing I noteiced is using string for constant comparation is bad practice in konfera.models.ticket_type there is PUBLIC constant which should be used in such case.